contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -208,6 +206,34 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         return streamingFee.add(profitFee);
     }
 
+    /*
+     * Validate then set new streaming fee.
+     *
+     * @param  _newFeePercentage       Fee type and new streaming fee encoded in bytes
```

Comment:
Variable name isn't that accurate, since its not just the percentage. Maybe it is `newFeeData`